Rework the How_to_Use_the_Offset_Wizard guide (#5249)
ppy/osu-wiki: wiki/Guides/How_to_Use_the_Offset_Wizard/en.md

> # How to use the offset wizard

Reviewer: ```suggestion
# How to use the Offset Wizard
```

> # How to use the offset wizard
> *See also: [Universal Offset](/wiki/Universal_offset)*

Reviewer: ```suggestion
# How to use the offset wizard

*See also: [Universal offset](/wiki/Universal_offset)*
```

> If the [Hitsounds](/wiki/Beatmapping/Hitsound) feel out of sync with the music, in every beatmap you play, you will more than likely need to change your [universal offset](/wiki/Universal_offset). The easiest way to do this is by using the **offset wizard**.

Reviewer 2: ```suggestion
If the [Hitsounds](/wiki/Beatmapping/Hitsound) felt slightly out of sync with the music, in every beatmap that you've played, you will more than likely need to change your [Universal offset](/wiki/Universal_offset). The easiest way to do this is by using the **offset wizard**.
```

Reviewer 3: I think the tense is fine as-is?

Author: Feels wrong changing to past tense so will leave as is. Fixing everything else though as suggested. Thanks for the help :)

Reviewer 2: Yeah, it is fine. But just wanted to make it to sound a little bit nicer (i think?)

> If the [Hitsounds](/wiki/Beatmapping/Hitsound) feel out of sync with the music, in every beatmap you play, you will more than likely need to change your [Universal Offset](/wiki/Universal_offset). The easiest way to do this is by using the **Offset Wizard**.

Reviewer: ```suggestion
If the [hitsounds](/wiki/Beatmapping/Hitsound) feel out of sync with the music, in every beatmap you play, you will more than likely need to change your [universal offset](/wiki/Universal_offset). The easiest way to do this is by using the **Offset Wizard**.
```

> Follow these steps to adjust your universal offset using the offset wizard:

Reviewer 2: ```suggestion
Follow these steps to adjust your Universal offset using the Offset wizard:
```

> Follow these steps to adjust your Universal Offset using the Offset Wizard:

Reviewer: ```suggestion
Follow these steps to adjust your universal offset using the Offset Wizard:
```

> **Opening the offset wizard**:

Reviewer 2: ```suggestion
**Opening the Offset wizard**:
```

1. Open osu!.
2. Go to the `Options` menu, or press `Ctrl` + `O`.
3. Search "offset" in the quick-search and click on `Offset wizard`.

> The offset wizard will then open, playing the song that was previously playing in osu!, while a green bar scrolls across the screen and a metronome plays.

Reviewer 2: ```suggestion
The  Offsetwizard will then be opened, playing the song that was previously playing in osu!, while a green bar scrolls across the screen and a metronome plays.

> The Offset Wizard will then open, playing the song that was previously playing in osu!, while a green bar scrolls across the screen and a metronome plays.

Reviewer: ```suggestion
The Offset Wizard will then open, playing the song that was previously played in osu!, while a green bar is scrolling across the screen and a metronome is ticking.
```

> **Using the offset wizard to calibrate your universal offset**:

Reviewer 2: ```suggestion
**Using the Offset wizard to calibrate your Universal offset**:
```

Reviewer 3: If you're gonna capitalise it, capitalise as `Universal Offset`, not just half.

> **Using the Offset Wizard to calibrate your Universal Offset**:

Reviewer: ```suggestion
**Using the Offset Wizard to calibrate your universal offset**:
```

> 1. Listen carefully to the song and the metronome playing together.

Reviewer: ```suggestion
1. Listen carefully to the song and the metronome ticking together.
```

Author: the song and the metronome dont "tick" together though, so i'm not sure this makes sense

> 2. Start adjusting the Universal Offset up or down, using the `Up arrow` or `Down arrow` keys, or the `Scroll wheel`.

Reviewer: ```suggestion
2. Start adjusting the universal fffset up or down, using the corresponding arrow keys, or the mouse scroll wheel.
```

>    - If, in-game, the hitsounds seem like they are playing too early, you will need to *decrease* your universal offset.

Reviewer 2: ```suggestion
   - If the hitsounds seem like they are playing too early in-game,  you will need to *decrease* your Universal offset.

>    - If the hitsounds seem like they are playing too early in-game, you will need to *decrease* your Universal Offset.
>    - If the hitsounds seem like they are playing too late in-game, you will need to *increase* your Universal Offset.

Reviewer: ```suggestion
   - If the hitsounds seem like they are playing too early in-game, you will need to *decrease* your universal offset.
   - If the hitsounds seem like they are playing too late in-game, you will need to *increase* your universal offset.
```

>    - If, in-game, the hitsounds seem like they are playing too late, you will need to *increase* your universal offset.

Reviewer 2: ```suggestion
   - If the hitsounds seem like they are playing too late in-game, you will need to *increase* your Universal offset.

> 3. Adjust it until you hear the metronome line up perfectly with the song, using the scrolling bar on the screen to help you.

Reviewer: ```suggestion
3. Adjust it until you hear the metronome line up perfectly with the song, using the scrolling bar on the screen to help you.
```

> 3. Adjust it until you hear the metronome line up perfectly with the song, using the scrolling bar on the screen to help you.

Reviewer 2: ```suggestion
Adjust it until you hear the metronome lines up perfectly with the song, using the scrolling bar on the screen to help you.
```

Author: "line" is fine as is, would be "lines" if the sentence was like "Adjust it until the metronome lines up perfectly with the song" , but I'd rather have the "until you hear" part still in.

> 4. Once you have successfully calibrated your Universal Offset, press the `Back` button at the bottom left to exit the Offset Wizard.

Reviewer: ```suggestion
4. Once you have successfully calibrated your universal offset, press the `Back` button at the bottom left to exit the Offset Wizard.
```

Author: Thanks, didnt know not to capitalize universal offset (im assuming because its not the topic of the article) fixed everything you suggested except one thing(which i have commented on) thanks again :)

> 4. Once you have successfully calibrated your universal offset, press the `Back` button in the bottom left to exit the offset wizard.

Reviewer 2: ```suggestion
4. Once you have successfully calibrated your Universal offset, press the `Back` button at the bottom left to exit the Offset wizard.
```

# How to use the Offset Wizard

> *See also: [Offset Wizard](/wiki/Options%2FOffset_Wizard) and [Universal Offset](/wiki/Universal_offset)*

Reviewer: ```suggestion
*See also: [Offset Wizard](/wiki/Options/Offset_Wizard) and [Universal Offset](/wiki/Universal_offset)*
```

If the [hitsounds](/wiki/Beatmapping/Hitsound) feel out of sync with the music, in every beatmap you play, you will more than likely need to change your [universal offset](/wiki/Universal_offset). The easiest way to do this is by using the **Offset Wizard**.

Follow these steps to adjust your universal offset using the Offset Wizard:

**Opening the Offset Wizard**:

1. Open osu!.
2. Go to the `Options` menu, or press `Ctrl` + `O`.
3. Search "offset" in the quick-search and click on `Offset wizard`.

The Offset Wizard will then open, playing the song that was previously played in osu!, while green bars scroll across the screen and a metronome is ticking.

> **Using the Offset Wizard to calibrate your universal offset**:

Reviewer: ```suggestion
## Calibrating the universal offset
```
